packages/api-client/index.ts

```diff
@@ -92,8 +92,15 @@ class APIClient {
       this.req("DELETE", `/api/v1/profile/delete_profile_picture`),
     getAllStudents: async (courseId: number): Promise<any> =>
       this.req("GET", `/api/v1/profile/${courseId}/id`, undefined),
-    getStudent: async (sid: number): Promise<any> =>
-      this.req("GET", `/api/v1/profile/${sid}/student`, undefined),
+    getStudent: async (id: number): Promise<any> =>
```

Comment:
would it make sense to use `sid` parameter instead of `id`?

Reply:
sid is the student number and is not a primary key